aiorwlock/__init__.py

```diff
@@ -17,15 +17,31 @@
 # implementation based on:
 # http://bugs.python.org/issue8800
 
+_global_lock = threading.Lock()
+
+
+class _LoopBoundMixin:
+    _loop = None
+
+    def _get_loop(self) -> Loop:
+        loop = asyncio.get_event_loop()
```

Comment:
But, do link to the upstream code this is copied from:
```suggestion
        """From: https://github.com/python/cpython/blob/main/Lib/asyncio/mixins.py"""
        loop = asyncio.get_event_loop()
```

Reply:
Done in 76c21a551cc30cccba21f73525dceef20766594a.